### PR TITLE
[Fix](nereids) remove the uppper filter after push predicates into external scan

### PR DESCRIPTION
**Pull request by feiniaofeiafei**

### CODE REVIEW

**Comment by feiniaofeiafei** [comment]
run buildall

**Comment by feiniaofeiafei** [comment]
run p0

**Comment by morrySnow** [comment]
add desc

File: fe/fe-core/src/main/java/org/apache/doris/nereids/rules/rewrite/PushConjunctsIntoJdbcScan.java [hunk 1]
```diff
@@ -32,8 +32,7 @@ public Rule build() {
         return logicalFilter(logicalJdbcScan()).thenApply(ctx -> {
             LogicalFilter<LogicalJdbcScan> filter = ctx.root;
             LogicalJdbcScan scan = filter.child();
-            LogicalJdbcScan rewrittenScan = scan.withConjuncts(filter.getConjuncts());
-            return new LogicalFilter<>(filter.getConjuncts(), rewrittenScan);
+            return scan.withConjuncts(filter.getConjuncts());
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/rules/rewrite/PushConjunctsIntoJdbcScan.java:35]: it is right? maybe some conujuncts could not be processed by scan?

**Comment by zy-kkk** [comment]
The modification made in this PR is incorrect. When pushing predicates into the external scan, `JdbcScanNode` will further determine which predicates can be pushed down and may only push down a portion of the predicates. Therefore, it is necessary to retain the complete filter to ensure that all predicates are applied after fetching the data.

Please revert the change to retain the upper filter after pushing predicates into the external scan.

**Comment by feiniaofeiafei** [comment]
run p0

**Pull request closed without merging**